ledger/src/bank_forks_utils.rs

```diff
@@ -139,6 +139,7 @@ pub fn load_bank_forks(
                 process_options,
                 cache_block_meta_sender,
                 accounts_update_notifier,
+                snapshot_config,
```

Comment:
What I had in mind is that we'd take the snapshot of a frozen bank 0 in `solana-genesis` (and in the similar looking code in `test-validator/src/lib.rs` that establishes the test validator genesis config).  

If we did that then `load_bank_forks()` could be simplified into always loading a snaphost and the hacky `blockstore_processor::process_blockstore_for_bank_0` thing only lives in `solana-genesis`

Reply:
Ah I gotcha, I like that!